kas/imx93-frdm.yml

```diff
@@ -0,0 +1,19 @@
+header:
+  version: 11
+  includes:
+    - kas/base.yml
+    - kas/vendor/nxp-frdm.yml
+    - kas/vendor/freescale.yml
+    - kas/target/complete.yml
+
+repos:
+  meta-avocado:
+    path: meta-avocado
+    layers:
+      meta-avocado-nxp:
```

Comment:
6 layers of BitBake excluding the usual suspects. Quite the challenge. Thanks, NXP!